### PR TITLE
🔨 Add error if source is invalid & ✏️ Put same typos

### PR DESCRIPTION
**Pull request by dezel-2109**
Hello,
I add error if the source is invalid (if source is 0).
And I change `playerId` by `playerSrc`.

(And I verify everything and it's work)

### CODE REVIEW

**Review by CsokiHUN — APPROVED** [review]

File: src/api/player/server/getPlayerName.lua [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-        end
+function API_Player:getName(playerSrc)
+    if (not playerSrc or playerSrc <= 0) then
+        return NCS:die('Unable to find name : source is invalid')
+    end
+    ---@type string
+    local name <const> = GetPlayerName(playerSrc)
+    if (name) then
+        return (name)
     end
```
Comment (MoskalykA) [inline src/api/player/server/getPlayerName.lua:13]:
```suggestion
    return GetPlayerName(playerSrc)
```

File: src/api/player/server/getPlayerIdentifier.lua [hunk 2]
```diff
@@ -3,5 +3,8 @@
 ---@return table
 ---@public
 function API_Player:getIdentifier(playerSrc)
+    if (not playerSrc or playerSrc <= 0) then
+        return NCS:die('Unable to find identifier : source is invalid')
```
Comment (MoskalykA) [inline src/api/player/server/getPlayerIdentifier.lua:7]:
```suggestion
        return NCS:die("Unable to find identifier : source is invalid")
```

File: src/api/player/server/getPlayerIdentifiers.lua [hunk 3]
```diff
@@ -3,11 +3,14 @@
 ---@return table
 ---@public
 function API_Player:getIdentifiers(playerSrc)
+	if (not playerSrc or playerSrc <= 0) then
+		return NCS:die('Unable to find identifiers : source is invalid')
```
Comment (MoskalykA) [inline src/api/player/server/getPlayerIdentifiers.lua:7]:
```suggestion
		return NCS:die("Unable to find identifiers : source is invalid")
```

File: src/api/player/server/getPlayerLicense.lua [hunk 4]
```diff
@@ -1,9 +1,12 @@
 ---getLicense
----@param playerId number
+---@param playerSrc number
 ---@return string
 ---@public
-function API_Player:getLicense(playerId)
-    for _, v in pairs(GetPlayerIdentifiers(playerId)) do
+function API_Player:getLicense(playerSrc)
+    if (not playerSrc or playerSrc <= 0) then
+        return NCS:die('Unable to find license : source is invalid')
```
Comment (MoskalykA) [inline src/api/player/server/getPlayerLicense.lua:7]:
```suggestion
        return NCS:die("Unable to find license : source is invalid")
```

File: src/api/player/server/getPlayerName.lua [hunk 5]
```diff
@@ -1,14 +1,14 @@
...
-function API_Player:getName(playerId)
-    if (playerId) then
-        ---@type string
-        local name <const> = GetPlayerName(playerId)
-        if (name) then
-            return (name)
-        end
+function API_Player:getName(playerSrc)
+    if (not playerSrc or playerSrc <= 0) then
+        return NCS:die('Unable to find name : source is invalid')
```
Comment (MoskalykA) [inline src/api/player/server/getPlayerName.lua:7]:
```suggestion
        return NCS:die("Unable to find name : source is invalid")
```

File: src/api/player/server/kickPlayer.lua [hunk 6]
```diff
@@ -4,6 +4,9 @@
 ---@return void
 ---@public
 function API_Player:kickPlayer(playerSrc, reason)
+    if (not playerSrc or playerSrc <= 0) then
+        return NCS:die('Specified source is invalid : kick canceled')
```
Comment (MoskalykA) [inline src/api/player/server/kickPlayer.lua:8]:
```suggestion
        return NCS:die("Specified source is invalid : kick canceled")
```

**Review by MoskalykA — CHANGES_REQUESTED** [review]

**Comment by dezel-2109** [comment]
Good!

**Review by MoskalykA — APPROVED** [review]

**Pull request merged**